repo_tests/ansible_manifest_resolution_contrast_test.py

```
@@ -0,0 +1,214 @@
+
+# --- one_manifest_findings -------------------------------------------------
+
+_FILE_ONE = "roles/alpha/tasks/main.yml"
+_FILE_TWO = "playbooks/alpha.yml"
+_CLEAN = dict(
+    shapes={_VENV: _MANIFEST},
+    bindings={_VENV: frozenset({_ALPHA})},
+    per_file={_FILE_ONE: frozenset({_ALPHA}), _FILE_TWO: frozenset({_ALPHA})},
+    filling=frozenset({_FILE_ONE, _FILE_TWO}),
+    expected=_ALPHA,
+)
```

Ruff (0.16.3)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)